### DISCUSSION TITLE
Rename VACUUM command

### DISCUSSION THREAD

**Discussion by Guba** [pawelsalawa/sqlitestudio#4260] [General]
I'm new to SQLite, so when I was looking for a feature to optimize a database I did not know that this feature is called "vacuum" in SQLite-ese language. 
I wrongly thought that the term "vacuum", that means "emptyness", was indicating a feature to empty a database in a single shot.
Instead it means "optimization", but I had to look for the optimizion concept on the internet to find this incomprehensible term.
I propose to change the term "vacuum" with the term "optimization (vacuum)" for a better readability.
I'm also new on this GitHub, and I'm here not to creating an issue, but for proposing a new feature, as I clicked on the "Propose new feature" into the SQLite Studio HELP menu. 
Also this should be changed, because this is not a problem but a proposal.
So the should be a correct link to the right page in Git to propose new features.

**Comment by pawelsalawa** [comment]
Well, this is how it's called in the SQLite itself. SQLiteStudio just stays in sync with how it's called in SQLite.

If you want this to be renamed, you probably need to raise it to SQLite developers.

**Comment by yfdyh000** [comment]
It is not an optimization, it is just a file compaction, removing the gap that marked as idle. It should be understood as a verb, i.e. use a vacuum cleaner.

The [optimize pragma](https://www.sqlite.org/lang_analyze.html) is already existing in SQLite, it considers calls `ANALYZE;` command to optimize the query performance.
